packages/components/src/components/telekom/telekom-mobile-menu-item/telekom-mobile-menu-item.css

```diff
@@ -99,4 +99,5 @@
 :host::part(icon-right-container) {
   display: flex;
   align-items: center;
+  position: absolute;
```

Comment:
I wonder why is this needed, I couldn't see the effect 👀

Reply:
<img width="499" alt="Screenshot 2023-12-20 at 15 03 16" src="https://github.com/telekom/scale/assets/97440128/ec4d57e8-c5a5-4cd2-97f1-1dff15bf1cc4">

without it the focus outline doesn't include the icon...